db/migrate/20220127173633_add_users_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] There is an issue with a user foreign key that is not available in `posts` table,  please fix that
![fr](https://user-images.githubusercontent.com/44745855/151519487-5dedffdf-5cab-459b-a35b-e1fd2e2975f5.png)

Reply:
you need to remove all the migrations but not the schema